packages/byods/src/token-manager/index.ts

```diff
@@ -1,17 +1,19 @@
 import fetch, {Response} from 'node-fetch';
 
 import {APPLICATION_ID_PREFIX, PRODUCTION_BASE_URL} from '../constants';
-import {TokenResponse, OrgServiceAppAuthorization, ServiceAppAuthorizationMap} from '../types';
+import {TokenResponse, OrgServiceAppAuthorization} from '../types';
+import {TokenStorageAdapter} from '../token-storage-adapter/types';
+import {InMemoryTokenStorageAdapter} from '../token-storage-adapter/inMemoryTokenStorage';
 
 /**
  * The token manager for the BYoDS SDK.
  */
 export default class TokenManager {
-  private serviceAppAuthorizations: ServiceAppAuthorizationMap = {};
```

Comment:
Now that we have a token storage adapter, we no longer require the map. The adapter will hold the references of the tokens and org id.